scripts/wattlab_parity_ofdd_rust_bundle.py

```
@@ -0,0 +1,401 @@
+        rc = capture_main()
+        if rc not in (0, 4):
+            return rc
+        token = login(args.base, args.admin_password) if args.admin_password else None
+        capture_extra(args.base, token, args.building_id, args.capture_out)
+    else:
+        extra_path = args.capture_out / "extra_analytics.json"
+        if not extra_path.is_file():
+            try:
+                token = login(args.base, args.admin_password) if args.admin_password else None
+                capture_extra(args.base, token, args.building_id, args.capture_out)
+            except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)